### PR TITLE
Cmake moderization

### PR DESCRIPTION
**Pull request by MohammadAlTurany**
Remove FindPackages for all packages that deliver meanwhile a CMakePackage.Config (e.g: G3, G4, VGM, G4VMC, ..etc)

### CODE REVIEW

File: geane/CMakeLists.txt [hunk 1]
```diff
@@ -27,8 +27,8 @@ ${CMAKE_SOURCE_DIR}/alignment
 
 Set(SYSTEM_INCLUDE_DIRECTORIES
   ${ROOT_INCLUDE_DIR} 
-  ${GEANT3_INCLUDE_DIR}
   ${Geant3_INCLUDE_DIRS}
+  ${Geant3_INCLUDE_DIRSS}
```
Comment (fuhlig1) [inline geane/CMakeLists.txt]: Do we really have a variable Geant3_INCLUDE_DIRSS or is this a typo?

File: cmake/scripts/config.sh.in [hunk 2]
```diff
@@ -8,41 +8,24 @@ if [ $same_system -eq 1 ]; then
...
+		export Geant4VMC_LIBRARY_DIR="@Geant4VMC_LIBRARY_DIR@"
+		export Geant4VMC_MACRO_DIR="@Geant4VMC_MACRO_DIR@"
 		export PLUTO_LIBRARY_DIR="@PLUTO_LIBRARY_DIR@"
 		export PLUTO_INCLUDE_DIR="@PLUTO_INCLUDE_DIR@"
 		export PYTHIA6_LIBRARY_DIR="@PYTHIA6_LIBRARY_DIR@"
-		export G3SYS="@G3SYS@"
-		export GEANT3_INCLUDE_DIR="@GEANT3_INCLUDE_DIR@"
-		export GEANT3_LIBRARY_DIR="@GEANT3_LIBRARY_DIR@"
-		export GEANT3_LIBRARIES="@GEANT3_LIBRARIES@"
+		export Geant3_INCLUDE_DIRS="@Geant3_INCLUDE_DIRS@"
```
Comment (fuhlig1) [inline cmake/scripts/config.sh.in]: I think G3SYS is missing.

File: cmake/scripts/config.csh.in [hunk 3]
```diff
@@ -3,45 +3,30 @@
...
+		setenv Geant4VMC_LIBRARY_DIR "@Geant4VMC_LIBRARY_DIR@"
+		setenv Geant4VMC_MACRO_DIR "@Geant4VMC_MACRO_DIR@"
 		setenv PLUTO_LIBRARY_DIR "@PLUTO_LIBRARY_DIR@"
 		setenv PLUTO_INCLUDE_DIR "@PLUTO_INCLUDE_DIR@"
 		setenv PYTHIA6_LIBRARY_DIR "@PYTHIA6_LIBRARY_DIR@"
-		setenv G3SYS "@G3SYS@"
-		setenv GEANT3_INCLUDE_DIR "@GEANT3_INCLUDE_DIR@"
-		setenv GEANT3_LIBRARY_DIR "@GEANT3_LIBRARY_DIR@"
-		setenv GEANT3_LIBRARIES "@GEANT3_LIBRARIES@"
+		setenv Geant3_INCLUDE_DIRS "@Geant3_INCLUDE_DIRS@"
```
Comment (fuhlig1) [inline cmake/scripts/config.csh.in]: I think G3SYS is missing.

File: cmake/modules/FindPLUTO.cmake [hunk 4]
```diff
@@ -8,10 +8,10 @@
 # - Try to find PLUTO instalation
 # Once done this will define
 #
```
Comment (fuhlig1) [inline cmake/modules/FindPLUTO.cmake]: Geant3 should be exchanged by Pluto. This is an really old cut-n-paste error.

File: cmake/scripts/config.sh.in (hunk 2 above)
Comment (MohammadAlTurany) [reply cmake/scripts/config.sh.in]: Yes, Thanks!

File: geane/CMakeLists.txt (hunk 1 above)
Comment (MohammadAlTurany) [reply geane/CMakeLists.txt]: Typo

File: cmake/scripts/config.csh.in (hunk 3 above)
Comment (MohammadAlTurany) [reply cmake/scripts/config.csh.in]: Yes, I added it

File: cmake/modules/FindPLUTO.cmake (hunk 4 above)
Comment (MohammadAlTurany) [reply cmake/modules/FindPLUTO.cmake]: done!

**Pull request merged**